src/Processors/Formats/Impl/JSONColumnsBlockInputFormatBase.cpp

```diff
@@ -128,7 +128,7 @@ Chunk JSONColumnsBlockInputFormatBase::generate()
         {
             /// Check if this name appears in header. If no, skip this column or throw
             /// an exception according to setting input_format_skip_unknown_fields
-            if (!name_to_index.has(*column_name))
+            if (!name_to_index.count(*column_name))
```

Comment:
Indeed, nice catch. Changed to `find` (I used `count` here in the first place because I was searching for something more similar to `contains` in terms of writing, which is missing in sparsehash, nevermind)